packages/expo-react-native-adapter/src/SyntheticPlatformEmitter.ts

```diff
@@ -0,0 +1,18 @@
+import RCTDeviceEventEmitter from 'react-native/Libraries/EventEmitter/RCTDeviceEventEmitter';
+import EventEmitter from 'react-native/Libraries/vendor/emitter/EventEmitter';
+
+/*
+ * This emitter is used for sending synthetic native events to listeners
+ * registered in the API layer with `NativeEventEmitter`.
+ */
+class SyntheticPlatformEmitter extends EventEmitter {
+  constructor() {
+    super(RCTDeviceEventEmitter.sharedSubscriber);
```

Comment:
I had a thought -- since we want this emitter only to emit events (no subscribing), what if it were more like this:

```js
class SPE {  // does not extend EE and therefore has none of its methods
  _emitter = new EventEmitter(RCTDEE.ss);

  emit(...) {
    this._emitter.emit(...);
  }
}
```